alpenhorn/auto_import.py

```diff
@@ -44,17 +44,21 @@ def _import_file(node, root, acq_name, file_name):
     """
     # global import_done
     curr_done = True
-    fullpath = "%s/%s/%s" % (root, acq_name, file_name)
+    fullpath = os.path.join(root, file_path)
     log.debug("Considering %s for import." % fullpath)
 
     # Skip the file if ch_master.py still has a lock on it.
-    if os.path.isfile("%s/%s/.%s.lock" % (root, acq_name, file_name)):
+    dir_name = os.path.dirname(file_path)
+    base_name = os.path.basename(file_path)
+    if os.path.isfile(os.path.join(root, dir_name, ".%s.lock" % base_name)):
         log.debug("Skipping \"%s\", which is locked by ch_master.py." % fullpath)
         return
 
     # Check if we can handle this acquisition, and skip if we can't
-    if ac.AcqType.detect(acq_name, node) is None:
-        log.info("Skipping non-acquisition path %s." % acq_name)
+    # TODO: detect on the full file_path or just dir_name?
```

Comment:
I think the full file path is the most natural thing here, i.e. the purpose is to detect what type of acq this is given the file we are importing.